lib/fp.js

```diff
@@ -37,23 +34,21 @@ const compose = (
   return res;
 };
 
-const maybe = (
-  // Apply given function to value or default value
-  fn, // function
-  defVal, // default value
-  value // value (optional)
-  // Returns: result of `fn` or `defVal`
-) => (
+// Apply given function to value or default value
+//   fn - function
+//   defVal - default value
+//   value - value (optional)
+// Returns: result of `fn` or `defVal`
+const maybe = (fn, defVal, value) => (
   value !== undefined && value !== null ? fn(value) : defVal
 );
 
-const zip = (
-  // Zip several arrays into one
-  ...arrays // array of arrays,
-  // Returns: array, length is minimal of input arrays length
-  // Element with index i of resulting array is array with
-  // elements with index i from input arrays
-) => {
+// Zip several arrays into one
+//   arrays - array of arrays,
+// Returns: array, length is minimal of input arrays length
+// Element with index i of resulting array is array with
```

Comment:
The description should be at the top. (Also applies in other places)

Reply:
As for me it belongs more to `Returns` than to function description

Reply:
Oh, I thought it's not about the return value. Could you then fix the grammar of this sentence / add punctuation to make it a whole sentence please.